backend/migrations/001_initial.sql

```diff
@@ -0,0 +1,146 @@
+-- +goose Up
+
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS Organisations (
+    id INT NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(150),
+    email VARCHAR(150) NOT NULL,
+    githubID VARCHAR(150) NOT NULL,
+    avatarUrl VARCHAR(300),
+    webSite VARCHAR(300),
+    `description` VARCHAR(300),
+
+    createdAt TIMESTAMP,
+    updatedAt TIMESTAMP,
+    deletedAt TIMESTAMP,
+
+    UNIQUE(email, githubID),
+    PRIMARY KEY(Id)
+);
+
+CREATE TABLE IF NOT EXISTS Users (
+    id INT NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(150),
+    email VARCHAR(150) NOT NULL,
+    githubID VARCHAR(150) NOT NULL,
+    avatarUrl VARCHAR(300),
+
+    createdAt TIMESTAMP,
+    updatedAt TIMESTAMP,
+    deletedAt TIMESTAMP,
+
+    UNIQUE(email, githubID),
+    PRIMARY KEY(Id)
+);
+
+CREATE TABLE IF NOT EXISTS Wallets (
+    id INT NOT NULL AUTO_INCREMENT,
+
+    walletID VARCHAR(300) NOT NULL,
```

Comment:
Why there is an id and WalletID ?

Reply:
Because the walledID is a long string key and the id is the identification of the wallet in the table !

Reply:
For me, i think, in the future a wallet should be represented like a short string key as a hash than a simple number.